api/.mvpstudio/kube/runit.yml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mvpstudio-web-api
+  namespace: stage-mvpstudio-web
+  labels:
+    app: mvpstudio-web-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mvpstudio-web-api
+  template:
+    metadata:
+      labels:
+        app: mvpstudio-web-api
+    spec:
+      containers:
+      - name: api-server
+        image: mvpstudio/mvpstudio-web-api
+        resources:
+           limits:
+              cpu: 0.25
+              memory: 250Mi
+           requests:
+              cpu: 0.25
+              memory: 250Mi
+        ports:
+        - containerPort: 8000
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1 
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mvpstudio-web-api
+
+  namespace: stage-mvpstudio-web
+  annotations:
+    getambassador.io/config: |
+      ---
+      apiVersion: ambassador/v0
+      kind: Mapping
+      name: mvpstudio-web-api-mapping
+      host: mvpstudio.org
```

Comment:
This host would take over our current website, so we want to have in a staging location. The url scheme for our staging apps is: https://appname.stage.apps.mvpstudio.org
In this case we could do: https://mvpstudio.stage.apps.mvpstudio.org
We have routing and ssl setup for all subdomains of *.stage.apps.mvpstudio.org so it will all work automatically.

Reply:
Actually, it's more than that I think. We need some routes served by the existing container and some by this new one so the `prefix` line below here needs to be something like `/api`. I'm not sure if we have to update the ambassador config for the existing one to say do **not** serve `/api` or not - can't recall how Ambassador merges things.